strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/io/ArtifactInputStream.java

```diff
@@ -152,6 +177,10 @@ public int read(byte[] bytes,
                 MessageDigest digest = (MessageDigest) entry.getValue();
                 digest.update(bytes, off, numberOfBytesRead);
             }
+            if(isInputStreamValid()){
```

Comment:
Cosmetic: We use braces on new lines. Could you please change this?

Reply:
Fixed